vuepress/docs/v6.3.2/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -38,76 +39,85 @@ npm install -g generator-jhipster-entando@6.3.2
+    - `Do you want to add validation rules to your field?` (No)
+    - `Do you want to add a field to your entity?` (Yes)
+    - `What is the name of your field?` ("location")
+    - `What is the type of your field?` (String)
+    - `Do you want to add validation rules to your field?` (No)
+    - `Do you want to add a field to your entity?` (No)
+    - `Do you want to add a relationship to another entity?` (No)
+    - `Do you want to use separate service class for your business logic?` (Up to you)
+       - If yes:
+       - `Do you want to use a Data Transfer Object (DTO)?` (Up to you)
+       - `Do you want to add filtering?` (Up to you)
+    - `Is this entity read-only?` (Up to you)
+    - `Do you want pagination on your entity?` (Yes, with infinite scroll)
+
+- If you asked to be prompted about micro frontends when generating the project:  
+    - `Do you want to generate micro frontends?` (Up to you)
+
+- At this point the blueprint will generate controllers, repositories, services, and micro frontends for your entity.  
+    - `Overwrite src/main/resources/config/liquibase/master.xml?` When prompted with a conflict at this stage, enter `a` for "All." This will override existing files with the configuration changes needed for your new entity.
+
+You now have an Entando project, including a Spring Boot microservice with database integration and React-based micro frontends.      
 
 ### Project Structure
-   * ```/src/main/docker``` contains Docker files to help with local development environments
-   * ```/src/main/java``` and ```src/main/resources``` contain the microservice codebase and configuration
-   * ```/ui``` holds the React-based micro frontends. By default each entity gets an MFE for details, form, and table.
-   * ```/bundle``` is used to assemble the project code into an Entando bundle.
+   * ```/src/main/docker``` contains Docker files to help with local development environments.
+   * ```/src/main/java``` and ```src/main/resources``` contain the microservice codebase and configuration.
+   * ```/ui``` holds the React-based micro frontends. By default, each entity is allocated an MFE for details, form, and table.
+   * ```/bundle``` is used to assemble the project code into an Entando Bundle.
 
 ## Next Steps
 You now have a choice:
-   - [Build your Entando bundle and deploy your microservice and micro frontends to the Entando Component Repository.](../pb/publish-project-bundle.md)
-   - [Go to the Running Locally tutorial to run your micro frontends and microservice in your local dev environment.](./run-local.md)
-   - [Learn about the key elements included in the Blueprint generated widgets](../../../docs/create/blueprint-features.md)
-   - [Iterate on your data model using the JHipster Domain Language (JDL)](./update-data-model.md)
+
+- [Build your Entando Bundle](../pb/publish-project-bundle.md) and deploy your microservice and micro frontends to the Entando Component Repository
```

Comment:
How about "Build and Deploy your Entando bundle to the Entando ..." because this makes it sound like deploying it will be a separate step. Another confusing thing is that a bundle is not really a bundle since the MS and MFE are deployed separately.

Reply:
@jyunmitch "a bundle is not really a bundle"?? please rephrase?

Reply:
After you brought this up and Nathan clarified it, it all makes sense. But if it is a bundle, they should stay together when deployed but sadly they don't. Nothing is wrong about the verbiage. I just noticed it always says it this way, 'deploy your mfe and ms...' and in thinking about it, it makes sense.